packages/in_app_purchase/in_app_purchase_storekit/darwin/Classes/InAppPurchasePlugin.m

```diff
@@ -270,62 +246,61 @@ - (void)presentCodeRedemptionSheetWithError:
 #endif
 }
 
-- (void)retrieveReceiptData:(FlutterMethodCall *)call result:(FlutterResult)result {
-  FlutterError *error = nil;
-  NSString *receiptData = [self.receiptManager retrieveReceiptWithError:&error];
-  if (error) {
-    result(error);
-    return;
+- (NSString *_Nullable)retrieveReceiptDataWithError:
```

Comment:
Here you should be using `nullable`: https://blog.google/technology/developers/gemma-open-models/

The one below is fine because it's complicated and needs the underscore versions.  `nullable` and `nonnull` should be preferred when available.

Reply:
i think you may have linked the wrong thing but I believe you :P

Reply:
here it is =) https://google.github.io/styleguide/objcguide.html#nullability